modular_skyrat/modules/modular_items/code/modular_clothing.dm

```diff
@@ -43,3 +43,97 @@
+
+/obj/item/clothing/suit/space/breaker
+	name = "cutter's space suit"
+	desc = "A pressure-sealed suit adorned with high-vis strips and plenty of gear rigging. A worn-out tag on the side says " + span_engradio("\"Safety Second\"") + "."
+	icon = 'modular_skyrat/master_files/icons/obj/clothing/suits.dmi'
+	worn_icon = 'modular_skyrat/master_files/icons/mob/clothing/suit.dmi'
+	icon_state = "space_breaker"
+	allowed = list(/obj/item/flashlight, /obj/item/tank/internals)
+	cell = /obj/item/stock_parts/cell/high/plus
+
+/obj/item/clothing/suit/space/breaker/worn_overlays(mutable_appearance/standing, isinhands, icon_file)
+	. = ..()
+	if(!isinhands)
+		. += emissive_appearance(icon_file, "[icon_state]-emissive", alpha = src.alpha)
+	//This will apply to all of them perfectly, because its based off icon_state
+
+/obj/item/clothing/suit/space/breaker/alt
+	icon_state = "space_breaker_alt"
+	//Sadly, spacesuits already use alt-click, which would be used to reskin here. It'll be easier to just supply extras...
+
+/obj/item/clothing/suit/space/breaker/armored
+	desc = "A pressure-sealed suit adorned with high-vis strips and plenty of gear rigging; additionally, this one's been fitted with an armored vest. A worn-out tag on the side says " + span_engradio("\"Safety Second\"") + "."
+	icon_state = "space_breaker_armored"
+	armor = list(MELEE = 35, BULLET = 30, LASER = 30, ENERGY = 40, BOMB = 25, BIO = 100, FIRE = 80, ACID = 70, WOUND = 10)
+	/*For comparison, view the two references:
+	/obj/item/clothing/suit/space: armor = list(MELEE = 0, BULLET = 0, LASER = 0,ENERGY = 0, BOMB = 0, BIO = 100, FIRE = 80, ACID = 70)
+	/obj/item/clothing/suit/armor: armor = list(MELEE = 35, BULLET = 30, LASER = 30, ENERGY = 40, BOMB = 25, BIO = 0, FIRE = 50, ACID = 50, WOUND = 10)
+	*/
+
+
+//Having to re-define and re-code these is annoying, but worth it I swear
+#define MODE_NONE ""
+#define MODE_MESON "meson"
+#define MODE_TRAY "t-ray"
+#define MODE_OBJECT "object"
+/obj/item/clothing/glasses/meson/engine/breaker	//Special goggles that can toggle between Meson, T-Ray, and Object modes
+	name = "cross-spectrum scanner"
+	desc = "A specially made pair of goggles that utilize Meson Radioscopy, Terahertz scanning technology, and experimental 'gestalt analysis' to provide the wearer the choice of seeing a ship's Structure, Systems, or Objects. Warranty void if worn without protective softsuit. A worn-out tag on the side says " + span_engradio("\"Safety Second\"") + "."
+	modes = list(MODE_NONE = MODE_MESON, MODE_MESON = MODE_TRAY, MODE_TRAY = MODE_OBJECT, MODE_OBJECT = MODE_NONE)
+	range = 3
+
+/obj/item/clothing/glasses/meson/engine/breaker/toggle_mode(mob/user, voluntary)
+	mode = modes[mode]
+	to_chat(user, "<span class='[voluntary ? "notice":"warning"]'>[voluntary ? "You turn the goggles":"The goggles turn"] [mode ? "to [mode] mode":"off"][voluntary ? ".":"!"]</span>")
+	if(connection_images.len)
```

Comment:
```suggestion
	if(length(connection_images))
```

Reply:
this is all copy-pasted code, i had to overwrite the proc to add our object mode. I'll commit this tho.